src/main/java/com/schematica/world/storage/Schematic.java

```diff
@@ -86,6 +89,7 @@ public IBlockState getBlockState(@NotNull final BlockPos pos)
         final int z = pos.getZ();
         final Block block = BLOCK_REGISTRY.getObjectById(this.blocks[x][y][z]);
 
+        //todo: be aware of deprecation and possible removal
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1135)